### PR TITLE
Cisdp 3141 - update to phm input file stage & schema

### PR DESCRIPTION
**Pull request by izpickle**
## Pre-review checklist

I have ensured that:

- [ ] Code runs successfully in the development environment.
- [ ] New code is tested to prove that it meets the specificaiton. If not, justify reasoning.
- [ ] New code is documented using [type hinting](https://realpython.com/lessons/type-hinting/) and following the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) docstring format.
- [ ] New code follows the project naming standards.

### CODE REVIEW

File: cishouseholds/pipeline/mapping.py [hunk 1]
```diff
@@ -1434,6 +1434,158 @@
...
+        "PHM-FUP-HST-016": "cis_covid_vacine_type",
+        "PHM-FUP-HST-017": "cis_covid_vaccine_type_other",
+        "PHM-FUP-HST-018": "cis_covid_vaccine_number_of_doses",
+        "PHM-FUP-HST-019": "cis_covid_vaccine_date",
+        "PHM-FUP-HST-020": "flu_vaccination_received",
+        "PHM-FUP-HST-021": "flu_vaccine_date",
+        "PHM-FUP-HST-022": "been_outside_uk",
+        "PHM-FUP-HST-023": "been_outside_uk_last_country",
+        "PHM-FUP-HST-024": "been_outside_uk_last_return_date",
+        "PHM-FUP-C19-001": "PHM_think_had_respiratory_infection",
```
Comment (murphc1) [inline cishouseholds/pipeline/mapping.py:1514]: Remove PHM_ from the following variable names: 
"PHM_think_had_respiratory_infection"
"PHM_think_had_respiratory_infection_type"
"PHM_think_had_flu_onset_date"
"PHM_think_had_other_infection_onset_date"

File: cishouseholds/pipeline/mapping.py [hunk 2]
```diff
@@ -1434,6 +1434,158 @@
...
+        "BLOOD_TEST_BARCODE": "blood_sample_barcode",
+        "SWAB_TEST_BARCODE": "swab_sample_barcode",
+        "COLLEX_OPEN_DATE": "participant_completion_window_start_date_string",
+        "COLLEX_CLOSE_DATE": "participant_completion_window_end_date_string",
+        "PORTAL_ID": "portal_id",
+        "WINDOW_START_DATE": "participant_completion_window_start_date",
+        "WINDOW_CLOSE_DATE": "participant_completion_window_end_date",
+        "PARTICIPANT_WINDOW_ID": "participant_completion_window_id",
+        "submitted_at": "survey_completed_datetime",
+        "flushed": "survey_completion_status_flushed",
```
Comment (murphc1) [inline cishouseholds/pipeline/mapping.py:1587]: Need to include the following:
"launch_language_code": "form_language_launch"
"submission_language_code": "form_language_submitted"

File: cishouseholds/phm/lookup.py [hunk 3]
```diff
@@ -207,8 +63,8 @@
     "cis_covid_vaccine_type_other": {"type": "string"},
     "cis_covid_vaccine_number_of_doses": {"type": "string"},
     "cis_covid_vaccine_date": {"type": "string"},
-    "flu_vaccination_received": {"type": "string"},
-    "flu_vaccination_received_date": {"type": "string"},
+    "flu_vaccine_received": {"type": "string"},
+    "flu_vaccine_received_date": {"type": "string"},
     "been_outside_uk": {"type": "string"},
     "been_outside_uk_last_country": {"type": "string"},
     "been_outside_uk_last_return_date": {"type": "string"},
```
[the remark this answers is not in the document]
Comment (murphc1) [reply cishouseholds/phm/lookup.py:70]: Need to include the following:
"form_language_launch": {"type": "string"},
"form_language_submitted": {"type": "string"},

**Review by murphc1 — CHANGES_REQUESTED** [review]

File: cishouseholds/pipeline/mapping.py (hunk 2 above)
Comment (slamothe99) [reply cishouseholds/pipeline/mapping.py:1587]: added

File: cishouseholds/phm/lookup.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (slamothe99) [reply cishouseholds/phm/lookup.py]: deleted lookup.py

**Review by kamila-m — APPROVED** [review]

File: cishouseholds/phm/high_level_phm_transformations.py [hunk 4]
```diff
@@ -24,25 +25,5 @@ def high_level_phm_transformations(df: DataFrame, participant_data_df: DataFrame
...
-    df = assign_datetime_from_combined_columns(
-        df=df,
-        column_name_to_assign="swab_taken_datetime",
-        date_column="blood_taken_date",
-        hour_column="blood_taken_time_hour",
-        minute_column="blood_taken_time_minute",
-        am_pm_column="blood_taken_am_pm",
-    )
-    df = assign_column_uniform_value(df, "survey_response_dataset_major_version", 4)
-    # df = generic_processing(df)
```
[the remark this answers is not in the document]
Comment (slamothe99) [reply cishouseholds/phm/high_level_phm_transformations.py:44]: it was moved to phm_transformations

**Pull request merged**